terraform/ecs/cluster_autoscaling.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_appautoscaling_target" "ecs_target" {
```

Comment:
Can we disable autoscaling for notify server for now? It isn't architected to be horizontally scalable yet since it is listens on inputs from the relay via websocket. This will result in duplicate request processing.

Reply:
Left the autoscaling in but set 1/1 as targets